### PR TITLE
update MariaDB manually with CLI (bsc#1132852, SOC-9022)

### PR DESCRIPTION
**Pull request by csymons-suse**
Customer does not use GUI, needs CLI instructions

(cherry picked from commit c18171d623c324f93e2a3e97287c61395f47b6f8)

### CODE REVIEW

File: xml/soc-operations-maintenance.xml [hunk 1]
```diff
@@ -23,11 +23,10 @@
...
    If one of those &barcl;s is deployed, patches are installed on the
    nodes. Patches that do not require a reboot will not cause a service
-   interruption. If a patch (for example, a kernel
-   update) requires a reboot after the installation, services running on the
-   machine that is rebooted will not be available within &cloud;.
-   Therefore it is strongly recommended to install those patches during a
-   maintenance window.
+   interruption. If a patch (for example, a kernel update) requires a reboot
+   after the installation, services running on the machine that is rebooted
+   will not be available within &cloud;.  Therefore it is strongly recommended
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]:
```suggestion
will not be available within &cloud;. Therefore, we strongly recommended
```

File: xml/soc-operations-maintenance.xml [hunk 2]
```diff
@@ -1444,8 +1442,49 @@
     <para>
      Perform a rolling upgrade to &mariadb; following the instructions at <link
      xlink:href="https://mariadb.com/kb/en/library/upgrading-between-minor-versions-with-galera-cluster/">Upgrading
-     Between Minor Versions with Galera Cluster</link>. Each node must upgraded
-     individually so that the cluster is always operational.
+     Between Minor Versions with Galera Cluster</link>.
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]: minor versions should be lowercase, no?

File: xml/soc-operations-maintenance.xml [hunk 3]
```diff
@@ -1444,8 +1442,49 @@
...
+    </para>
+    <para>
+     In general, the process involves the following steps:
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]:
```suggestion
The process involves the following steps:
```

File: xml/soc-operations-maintenance.xml [hunk 4]
```diff
@@ -1444,8 +1442,49 @@
...
+    </para>
+    <substeps>
+     <step>
+      <para>
+       Stop &mariadb;
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]:
```suggestion
Stop the &mariadb; cluster.
```

File: xml/soc-operations-maintenance.xml [hunk 5]
```diff
@@ -1444,8 +1442,49 @@
...
+      </para>
+     </step>
+     <step>
+      <para>
+       Uninstall the old versions of &mariadb; and the Galera wsrep provider
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]: missing a full stop at the end here

File: xml/soc-operations-maintenance.xml [hunk 6]
```diff
@@ -1444,8 +1442,49 @@
...
+      </para>
+     </step>
+     <step>
+      <para>
+       Install the new versions
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]:
```suggestion
Install the new versions of &mariadb; and the Galera wsrep provider.
```

File: xml/soc-operations-maintenance.xml [hunk 7]
```diff
@@ -1444,8 +1442,49 @@
...
+      </para>
+     </step>
+     <step>
+      <para>
+       Change configuration options if necessary
+      </para>
+     </step>
+     <step>
+      <para>
+       Start &mariadb;
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]: missing full stop

File: xml/soc-operations-maintenance.xml [hunk 8]
```diff
@@ -1444,8 +1442,49 @@
...
+      </para>
+     </step>
+     <step>
+      <para>
+       Run <command>mysql_upgrade</command> with the
+       <literal>--skip-write-binlog</literal> option
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]: option. *

File: xml/soc-operations-maintenance.xml (hunk 3 above)
Comment (asettle) [reply xml/soc-operations-maintenance.xml]: Question about this - was there a request to include the generalised instructions even though we link to the proper set? Seems redundant to rewrite it.

File: xml/soc-operations-maintenance.xml [hunk 9]
```diff
@@ -1454,5 +1493,79 @@
...
+    </para>
+    <screen>crm resource unmanage galera</screen>
+    <para>
+     Or put the whole cluster into maintenance mode:
+    </para>
+    <screen>crm configure property maintenance-mode=true</screen>
+   </step>
+   <step>
+    <para>
+     Pick a node other than the one currently targeted by the loadbalancer and
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]:
```suggestion
Pick a node other than the one currently targeted by the load balancer and
```

File: xml/soc-operations-maintenance.xml [hunk 10]
```diff
@@ -1454,5 +1493,79 @@
...
+     stop &mariadb; on that node:
+    </para>
+    <screen>crm_resource --wait --force-demote -r galera -V</screen>
+   </step>
+   <step>
+    <para>
+     Perform updates:
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]:
```suggestion
Perform the following updates:
```

File: xml/soc-operations-maintenance.xml (hunk 3 above)
Comment (csymons-suse) [reply xml/soc-operations-maintenance.xml]: yes. Lots of comments on the bug and Rocket Chat suggestions too. The process apparently is confusing as it happens. Someone? suggested giving a high level view.

Comment (asettle) [reply xml/soc-operations-maintenance.xml]: Huh fair enough - makes sense then :)

**Pull request merged**